consensus/src/main/java/org/apache/iotdb/consensus/config/MultiLeaderConfig.java

```diff
@@ -253,6 +274,9 @@ public static class Builder {
       private int maxWaitingTimeForAccumulatingBatchInMs = 500;
       private long basicRetryWaitTimeMs = TimeUnit.MILLISECONDS.toMillis(100);
       private long maxRetryWaitTimeMs = TimeUnit.SECONDS.toMillis(20);
+      private long maxWalBufferSize = 10 * 1024 * 1024 * 1024L;
```

Comment:
we can use more simple names here such as `throttleDownThreshold` and `throttleUpThreshold`

Reply:
Get it~